### PR TITLE
Add mopeka pro check integration docs

### PR DESCRIPTION
**Pull request by spbrogan**
## Description:

**Related issue (if applicable):** fixes <https://github.com/esphome/feature-requests/issues/1445>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#2618

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Comment by spbrogan** [comment]
I don't understand the development process.  How do i get the code PR merged.  It has been ready for a couple of months.   I did update the code pr today since i was using the code on a new node and wanted it rebased on current dev.

**Comment by jesserockz** [comment]
Hi @spbrogan The development process is like this: myself or one of only a handful of others will review the code when they have time.
Don't worry too much about this PR going stale as when someone reviews the code PR they can reopen this one again for review and merging.

File: components/sensor/mopeka_ble.rst [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **battery_level** (*Optional*): The information for the battery percentage
+  sensor.  Sensor uses a standard CR2032 battery.
+
+  - **name** (**Required**, string): The name for the battery level sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
```
Comment (jesserockz) [inline components/sensor/mopeka_ble.rst]: The sensor base config has these already, no need to repeat here.
```suggestion
- **level** (*Optional*): The percentage of full for the tank sensor

  - All options from :ref:`Sensor <config-sensor>`.

- **distance** (*Optional*): The raw distance/depth of the liquid for the sensor in mm.

  - All options from :ref:`Sensor <config-sensor>`.

- **temperature** (*Optional*): The information for the temperature sensor.
  This temperature is on the sensor and is not calibrated to ambient temperature.

  - All options from :ref:`Sensor <config-sensor>`.

- **battery_level** (*Optional*): The information for the battery percentage
  sensor.  Sensor uses a standard CR2032 battery.

  - All options from :ref:`Sensor <config-sensor>`.
```

**Review by jesserockz — CHANGES_REQUESTED** [review]

**Review by jesserockz — APPROVED** [review]

File: components/sensor/mopeka_ble.rst [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+
+Configuration variables:
+------------------------
+
+- **mac_address** (**Required**, MAC Address): The MAC address of the Mopeka
+  device.
+
+- **tank_type** (*Optional*, Default: 20LB_V): The tank type the sensor is
+  measuring.
```
Comment (jesserockz) [inline components/sensor/mopeka_ble.rst]:
```suggestion
- **tank_type** (**Required**): The tank type the sensor is measuring. See below.
```

**Pull request merged**